### DISCUSSION TITLE
Timestamp in quickfix logs

### DISCUSSION THREAD

**Discussion by ifisasia** [quickfix-j/quickfixj#838] [Q&A]
I am trying to configure the logging for the quickfixj and I notice that the timestamp in logs is in UTC, how can I make this in local time?
I already have TimeZone set in the settings but still logging with UTC timestamps. Did i missed a setting for File logging?
Settings:
TimeZone=Asia/Singapore
StartTime=00:00:00
EndTime=23:59:59

FileIncludeMilliseconds=Y
FileIncludeTimeStampForMessages=Y

Example logs:
20240701-06:33:12.705: 8=FIX.4.4...
20240701-06:33:18.465: 8=FIX.4.4...
20240701-06:33:22.693: 8=FIX.4.4 ...

Expected timing is 20240701 14:33 ..

please help. 
thank you.

**Comment by chrjohn** [comment]
Which OS are you using ? I assume this is dependent on the time zone settings on your machine. My logs have timestamps in CEST for example.

**Reply by ifisasia** [reply]
hi chrjohn, I am using win 11 and ubuntu for testing. currently expecting the logs in GMT +8.
I have a separate application logs and the logs timestamp is ok(timstamps in GMT+8,same as the machine time) 
but the logs produce by FileLogFactory and its settings are in utc timestamps.

**Reply by chrjohn** [reply]
I had to look into the code to see what that settings does. I don't remember anyone using it. Indeed it does only print the timestamp in UTC.
I'd rather suggest to turn that settings off (which is the default) and configure your logging framework (log4j or slf4j for example) to print the timestamp.
